modevo-transform/pom.xml

```diff
@@ -87,6 +87,31 @@
 	</dependencies>
 	<build>
 		<plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.0</version>
+                <executions>
+                    <execution>
+                        <id>make-jar</id>
+                        <phase>compile</phase>
```

Comment:
I think that this is better in a previous phase, eg. `process-resources`. Put a more significant name, as this is not the module jar, but something that is necessary for compilation.

Reply:
@javiertuya I changed it to process-resources and it runs ok in my computer using the same goals as in GitHub. However, in GitHub it is not able to create the Jar file:

`Warning:  JAR will be empty - no content was marked for inclusion!`

The run was the following one:

https://github.com/giis-uniovi/modevo/actions/runs/8572462627

I returned it back to the compile phase for now, which seems to be running ok. Do I keep looking into it or de we keep it in compile?

Reply:
process-resources is the latest phase before compile, so, keep it as is